src/shell/grabs/menu/default.rs

```diff
@@ -348,8 +358,12 @@ pub fn window_items(
                     let _ = handle.insert_idle(move |state| {
                         let mut shell = state.common.shell.write().unwrap();
                         let seat = shell.seats.last_active().clone();
-                        let res =
-                            shell.menu_resize_request(&resize_clone, &seat, ResizeEdge::RIGHT);
+                        let res = shell.menu_resize_request(
+                            &state.common.config,
```

Comment:
Thanks! I would prefer if you could just pass the threshold value instead of the whole config? (Applies to all the shell and grab methods).